cola/themes.py

```diff
@@ -1,17 +1,21 @@
 """Themes generators"""
 from __future__ import absolute_import, division, print_function, unicode_literals
+import os
 
 from qtpy import QtGui
 
 from .i18n import N_
 from .widgets import defs
 from . import icons
 from . import qtutils
+from . import resources
 
 
 class EStylesheet(object):
     DEFAULT = 1
     FLAT = 2
+    CUSTOM = 3  # Files located in ~/.config/git-cola/themes/*.qss
+    SYSTEM = 4  # Application qt5ct (Qt5 Configuration Tool)
```

Comment:
I always considered `DEFAULT` the same as what `SYSTEM` does...

Reply:
Strange, I just updated Ubuntu from 20.04 to 22.04. In 20.04 I couldn't get Default to use system config. But now that I have updated to 22.04 and I don't see any differences. I removed from this patch